src/main/java/com/qa/hubspot/page/DealsPage.java

```diff
@@ -0,0 +1,16 @@
+package com.qa.hubspot.page;
+
+import com.qa.hubspot.base.BasePage;
+
+public class DealsPage extends BasePage{
+	
+	public DealsPage() {
+		System.out.println("Deals Page const....");
+	}
+	
```

Comment:
Where is the rest of the code?
Where is page locators?
Where is page actions?
Please complete this page class